tools/xsd-to-bigquery-schema/.pylintrc

```diff
@@ -0,0 +1,441 @@
+# This Pylint rcfile contains a best-effort configuration to uphold the
```

Comment:
Do we need this file? This repo has already a Python linter with a predefined configuration. This file seems to comply with the Google Python style guide, but still, I would remove it and rely on the configuration provided by the repo.

Reply:
Yes I agree, it won't be needed, got carried with the code since I was working on this separately then moved to the PSO repo. Just removed it now.